stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/MediaManager.kt

```diff
@@ -871,3 +879,11 @@ class MediaManagerImpl(
         microphone.cleanup()
     }
 }
+
+fun MediaStreamTrack.trySetEnabled(enabled: Boolean) {
+    try {
+        setEnabled(enabled)
+    } catch (t: Throwable) {
+        // No-op
```

Comment:
We should at least log the swallowed exception, btw why not use `safeCall` here?

Reply:
Good catch.